### ISSUE TITLE
Conditional logic for exposing funding options during enrollment

### ISSUE THREAD

**Issue by victoriasuwardiman** [ctoec/ecis-experimental#1116]
Consider not showing funding types until an age group is selected (because age group affects the funding types available). 

Determine if there are other areas where this could be helpful.

**Comment by RyanHansz** [comment]
@victoriasuwardiman Is it possible to auto-choose age group based on a child's birthday? Or are there scenarios where the provider still has to choose? 

If age group is always based on birthday, then we could just automatically choose it.

Otherwise I'd propose this placeholder text until age group is selected:
<img width="558" alt="Screen Shot 2020-05-22 at 2 49 54 PM" src="https://user-images.githubusercontent.com/63253539/82699907-82390580-9c3b-11ea-8de9-0c64fd87ce25.png">

**Comment by victoriasuwardiman** [comment]
Yes! Here is the ticket that Ashley opened awhile back to do that - #462. Looks like it's in the dev backlog. So maybe once we get that, this won't be a problem anymore.
